frontend/components/TableContainer/_styles.scss

```diff
@@ -6,14 +6,14 @@
   // Container is responsive design used when customFilters is rendered
   .container {
     display: grid;
-    grid-template-columns: 1fr auto auto; /* First column takes all remaining space */
-    grid-template-rows: auto auto; /* Two rows */
+    grid-template-columns: 1fr auto; /* First column takes all remaining space */
+    grid-template-rows: auto auto; /* Two rows for smaller screens*/
```

Comment:
This seems like a generic enough component that it'll affect multiple places. Are all of those places listed in the QA plan? Guessing this fixes rather than breaks things but triple-checking.

Reply:
This is only used on these two pages, another example of starting new patterns, having them modified before they are even implemented anywhere else.

Reply:
Thanks for flagging though, because I forgot to updating comment on it's use case. Commit incoming.